share/tools/web_config/js/colorutils.js

```diff
@@ -191,32 +191,32 @@ function master_color_for_color(color_str) {
     })
 }
 
-/* Given a color name, like 'normal' or 'red' or 'FF00F0', return an RGB color string (or empty
+/* Given a color name, like 'normal' or 'red' or '#FF00F0', return an RGB color string (or empty
    string) */
 function interpret_color(str) {
```

Comment:
Currently `interpret_color` is not being used anywhere. I took a stab at this problem in a different way (by using this function, for one). Please check out https://github.com/charego/fish-shell/commit/b0bb1d0917a9f89b90d8df57b39a25500fea0a03 and let me know what you think. In addition to fixing the missing colors that you found, I also make a naive attempt to handle fish colors specified like '555 brblack' (which is what the "fish default" theme uses for autosuggestions).

See how autosuggestion text is white vs. gray in these examples.

Current version | Fixed version
------------ | -------------
<img width="256" alt="default_theme_1" src="https://user-images.githubusercontent.com/5209296/85112584-72c9bf80-b1db-11ea-9f1e-6a8430300880.png"> | <img width="256" alt="default_theme_2" src="https://user-images.githubusercontent.com/5209296/85112585-73faec80-b1db-11ea-8325-1f84b166fe54.png">

Reply:
That's a nice approach and great that you were able to make additional improvements!  Shall I close out this PR so you can replace with yours?

Reply:
Let's leave your PR open for a bit longer in case others want to chime in. Depending on what happens, we might do that later. Thanks!